CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # PRQL Changelog
 
+## 0.4.2 — [unreleased]
+
+**Features**:
+
+- `from_text` function that supports JSON and CSV formats. (@aljazerzen, @snth)
```

Comment:
Thanks. @max-sixty also needs a mention as he actually wrote the original `from_csv` and I just did the renaming to `from_text`.

Reply:
True, my bad

Reply:
I did very little here! And I left it hanging for ages so I should get a negative call-out